build.yaml

```diff
@@ -1 +1,5 @@
 projectType: extension
+additionalConfigs:
+  jdk17:
+    testJdkTool: OPEN-JDK17
+    mavenAdditionalArgs: -DminVersion=4.6.0 -Dtest=none -DfailIfNoTests=false -DruntimeProduct=MULE_EE
```

Comment:
Couldn't you add additional configs for JDK8 and JDK11 (both of them with -DminVersion=4.3.0)?

Reply:
Definitely! Didn't know if I should or not. Will do it right away